plone/memoize/view.py

```diff
@@ -22,10 +20,9 @@ def memogetter(*args, **kwargs):
             request = getattr(instance, "request", None)
 
             annotations = IAnnotations(request)
```

Comment:
If we would step back from using ZCA IAnnotations here by just setting an attribute on the request (setattr) this would probably also speed things up - but I did not performance-test that in this case.

Reply:
I agree with you, anyway I have tons of test where I clear the cache doing a 
```python
view.request.__annotations__.clear()
```